packages/cra-template-seed/template/src/util/Constants.js

```diff
@@ -0,0 +1,17 @@
+export default {
+  REACT_QUERY: {
+    CODES: {
+      SUCCESS: 'success',
+      LOADING: 'loading',
+      ERROR: 'error',
+    },
```

Comment:
I'm not happy with that duplicated code to be honest - this feels like we would be building up an artificial layer around react-query

Reply:
have refactored in the last commits.